Neos.Neos/Classes/Domain/Service/WorkspaceService.php

```diff
@@ -83,6 +84,12 @@ public function getWorkspaceMetadata(ContentRepositoryId $contentRepositoryId, W
      */
     public function setWorkspaceTitle(ContentRepositoryId $contentRepositoryId, WorkspaceName $workspaceName, WorkspaceTitle $newWorkspaceTitle): void
     {
+        $workspaceMetadata = $this->loadWorkspaceMetadata($contentRepositoryId, $workspaceName);
+        if ($workspaceMetadata->title->equals($newWorkspaceTitle)) {
```

Comment:
This would be the fix I guess!?
```suggestion
        if ($workspaceMetadata?->title->equals($newWorkspaceTitle)) {
```